manager.py

```diff
@@ -79,28 +80,103 @@ def stop(self):
         self._rest_manager.remove_web_handler(self._handler)
         super().stop()
 
-    def get_blocks_structure(self):
-        """ Return file structure under 'blocks' resource """
-        blocks = []
+    def get_blocks_structure(self, branches = 0):
```

Comment:
Suggest renaming parameter to get_branch/show_branch, etc., and make it a boolean, it gives the impression that values other than 0 and 1 may have a meaning